subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5.h

```diff
@@ -40,21 +40,22 @@
 #include <hal/nrf_vreqctrl.h>
 #include "radio_nrf5340.h"
 #elif defined(CONFIG_SOC_SERIES_NRF54LX)
+#include <hal/nrf_ppib.h>
 #include "radio_nrf54lx.h"
 #elif defined(CONFIG_BOARD_NRF52_BSIM)
 #include "radio_sim_nrf52.h"
 #elif defined(CONFIG_BOARD_NRF5340BSIM_NRF5340_CPUNET)
 #include <hal/nrf_vreqctrl.h>
 #include "radio_sim_nrf5340.h"
 #elif defined(CONFIG_BOARD_NRF54L15BSIM_NRF54L15_CPUAPP)
+#include <hal/nrf_ppib.h>
 #include "radio_sim_nrf54l.h"
 #else
 #error "Unsupported SoC."
 #endif
 
```

Comment:
I meant adding here
```suggestion
#if defined(CONFIG_BOARD_NRF54L15BSIM_NRF54L15_CPUAPP)
#include <hal/nrf_ppib.h>
#endif
```
or
```suggestion
#if PPIB_PRESENT
#include <hal/nrf_ppib.h>
#endif
```